lib/componentDocs/build.js

```diff
@@ -0,0 +1,49 @@
+const fs = require('fs');
+const fastGlob = require('fast-glob').async;
+const { parse } = require('react-docgen');
+const { normaliseType, normaliseDefaultValue } = require('./normaliseProp');
+
+const normaliseData = ({ props = {}, ...rest }) =>
+  Object.keys(props).reduce(
+    (acc, prop) => ({
+      ...acc,
+      props: {
+        ...acc.props,
+        [prop]: {
+          ...props[prop],
+          defaultValue: normaliseDefaultValue(props[prop]),
```

Comment:
Any chance of using `lodash.mapValues` here? Feel like this kinda stuff can be intimidating to some people and it would only need to be a `devDependency`.

Reply:
It could be that my `lodash` skills aren't up to scratch, but looking at the docs I can't see how the code would clean up this code. Due to the depth of the update taking place there would still need to be a bunch of spreading no?